### PR TITLE
Update cloud-billing.rst

### PR DESCRIPTION
**Pull request by johndavidlugtu**
Updates to the documentation that is shown when customers click "See how billing works" inside the Cloud product.

Based on notes here: https://docs.google.com/document/d/13V7V3P7kdy78_20XK-uljaWnzvfzyUlRqtuF21oC5Vg/edit#heading=h.1anbwvx5sa64

### CODE REVIEW

File: source/manage/cloud-billing.rst [hunk 1]
```diff
@@ -4,10 +4,19 @@ Mattermost Cloud billing
 .. include:: ../_static/badges/allplans-cloud.rst
   :start-after: :nosearch:
 
-Mattermost Cloud is offered as a pay-as-you-go subscription service.
+From February 2023, Mattermost Cloud Professional and Enterprise plans are offered as an annual subscription service.
 
-You will be billed for registered users in your workspace. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they are not deactivated in the billing period.
+.. note:: 
+   If you have an existing Cloud Professional monthly subscription you can keep it as is and are not required to switch to an annual subscription. If you'd like to switch, please follow these steps.
```
Comment (justinegeffen) [inline source/manage/cloud-billing.rst:10]: @johndavidlugtu, are there steps for this and will these customers ever need to switch to annual?

File: source/manage/cloud-billing.rst [hunk 2]
```diff
@@ -4,10 +4,19 @@ Mattermost Cloud billing
...
 
-Mattermost Cloud workspaces can be used free forever with Cloud Free. You can upgrade to a monthly per-user subscription, with Mattermost Professional and Mattermost Enterprise. 
+To upgrade to a Professional plan within Mattermost, select **View plans** in the header. For Enterprise plans, you need to [contact sales](https://mattermost.com/contact-sales/).
```
Comment (johndavidlugtu) [inline source/manage/cloud-billing.rst:12]: @justinegeffen they can also do this in via the System Console

File: source/manage/cloud-billing.rst (hunk 1 above)
Comment (johndavidlugtu) [reply source/manage/cloud-billing.rst:10]: There will be a [section in the System Console as well as a dismissible global banner](https://github.com/mattermost/mattermost-webapp/pull/12028) nudging them to switch to annual.

Note that the CTA might change based on this [conversation](https://community.mattermost.com/private-core/pl/rgf441zqw7y6jnonihbkwmehjo) @justinegeffen

**Comment by justinegeffen** [comment]
@johndavidlugtu, I'm happy to approve and merge this. Do you want to do a final sweep and could you confirm if we're merging today or tomorrow? Thanks!

File: source/manage/cloud-billing.rst [hunk 3]
```diff
@@ -4,10 +4,23 @@ Mattermost Cloud billing
 .. include:: ../_static/badges/allplans-cloud.rst
   :start-after: :nosearch:
 
-Mattermost Cloud is offered as a pay-as-you-go subscription service.
+From February 2023, Mattermost Cloud Professional and Enterprise plans are offered as an annual subscription service.
```
Comment (johndavidlugtu) [inline source/manage/cloud-billing.rst:7]: @justinegeffen any thoughts on making this more specific, to say, February 2, 2023? I'm also open to keeping it as is

Comment (justinegeffen) [reply source/manage/cloud-billing.rst:7]: Works for me!

File: source/about/cloud-subscriptions.rst [hunk 4]
```diff
@@ -1,58 +1,41 @@
 Cloud subscriptions
 ===================
 
+From February 2023, Mattermost Cloud subscriptions are switching from a monthly subscription to an annual subscription. However, existing Cloud Professional monthly customers can maintain their monthly subscription.
```
Comment (justinegeffen) [inline source/about/cloud-subscriptions.rst:4]:
```suggestion
From 2 February 2023, Mattermost Cloud subscriptions are switching from a monthly subscription to an annual subscription. However, existing Cloud Professional monthly customers can maintain their monthly subscription.
```

**Comment by justinegeffen** [comment]
Thanks, @johndavidlugtu! This looks good to me. Ready to merge when you are. :)

**Pull request merged**